packages/forms-web-app/src/controllers/full-appeal/you-cannot-appeal.js

```diff
@@ -7,7 +7,7 @@ const getYouCannotAppeal = async (req, res) => {
 	const { appealDeadline, appealPeriod } = appeal.eligibility;
 
 	// Clear appeal object
-	req.session.appeal = null;
+	// req.session.appeal = null;
```

Comment:
unsure on the implications of this?
will any questions filled in remain set
does this impact the check on deadline that kicks you out? i.e. can you change your answer to deadline or will it block users from progressing

Reply:
I have remvoed the back link from this page and reinstated the appeal scrubbing

Reply:
did it break things?

Reply:
It did cause issues, especially in terms of trying to navigate to the decision date page!   It would be necessary to just wipe the decision date props but that could have had problems if the decisions date wasn't the issue, so would have required a few further checks and so fell outside the timebox we set.